website/docs/basics/blocks-faq.md

```diff
@@ -136,7 +136,7 @@ Easy. We have a method you can extend that limits your project's block to the on
 
 ```php
   // Limits the usage of only custom project blocks.
-  add_filter('allowed_block_types', [ $this, 'getAllBlocksList' ], 10, 2);
+  add_filter('allowed_block_types_all', [ $this, 'getAllBlocksList' ], 10, 2);
```

Comment:
We should probably leave a note about the depreciation.

Something like: If you have WordPress version < 5.8 the hook name changed, so use this instead...

Something to that effect.

Reply:
please revert all the changes from the legacy part of the docs. We never change that. And also add a not as Denis said about the WP version and old hook

Reply:
Made the changes, please review... thanks!